src/sentry/notifications/notification_action/action_validation.py

```diff
@@ -11,23 +13,26 @@
 from .types import BaseActionValidatorHandler
 
 
+def _get_integration(validated_data: dict[str, Any], provider: str) -> RpcIntegration:
```

Comment:
to avoid some circular dep issues; we could create `action_validator` folder, and have this in a utils method. then have a separate file per type; `Slack`, `MSTeams`, etc.

not too critical though, just could be a nice way to organize and reduce what's being imported per file.